src/code.sh

```
@@ -77,7 +77,8 @@ _generate_region_vcfs() {
 
         bcftools mpileup -d 8000 -f "$reference_fasta_name" "$input_bam_name" \
             -r "$chromPos" -a FORMAT/AD,FORMAT/DP -Ou | \
-            bcftools call -mv -Oz -o "$output_vcf"
+            bcftools call -mv -Ou | \
+            bcftools norm -f "$reference_fasta_name" -Oz -o "$output_bcf"
```

Shellcheck (0.10.0)

[warning] output_bcf is referenced but not assigned (did you mean 'output_vcf'?).

(SC2154)